### PR TITLE
Add Namibia

### PR DESCRIPTION
**Pull request by systemcatch**
After a lot of experimentation I've managed to get this working reliably. Namibia imports big amounts of power from South Africa and Zambia (no data for either yet) so it's CO2 intensity will be skewed somewhat.

/cc @alixunderplatz 

ref #606 

na

### CODE REVIEW

**Comment by corradio** [comment]
Have you ever seen coal and oil values become > 0?

**Comment by alixunderplatz** [comment]
@corradio I think this is something that might happen, but not very often. For example, when South Africa faces power shortages and load shedding and cannot supply Namibian demand. [On IEA](https://www.iea.org/statistics/statisticssearch/report/?country=NAMIBIA&product=electricityandheat&year=2015), it says 1500 GWh were generated by hydro, 27 GWh from oil and 7 GWh from coal in 2015. So fossil plants are operating barely. Wind and solar might be big enough to lower that generation even more.

**Comment by alixunderplatz** [comment]
@systemcatch Just noticed, there are connections with Botswana and Angola, too!
It's in that 132 kV section:
- Omaere - Ghanzi is between Namibia -> Botswana
- Efundja - Ondjiva is between Namibia -> Angola

I think the header "cross border supplies" means, these places are connected to and supplied from the side of the Namibian system. So probably no export towards Namibia, but some more export could be displayed to other countries.

**Comment by systemcatch** [comment]
It's weird that those aren't included under import/export though.

I've never seen the oil plants above 0. The coal plant often has -0.4 generation which is set to None by the parser.

**Comment by corradio** [comment]
Maybe set coal to max(0, value) to make sure it is allowed to be >0 ?

**Comment by systemcatch** [comment]
So you'd rather coal be 0.0 rather than None if it's negative?

**Comment by corradio** [comment]
If it's only -0.45 we can assume it to be 0 I think? My main issue with
setting to None is to miss it the day it actually turns on.

**Comment by systemcatch** [comment]
When it does turn on it will not be None. It is only set to None if
negative or unreadable.

**Comment by corradio** [comment]
Perfect. That's even better.

File: parsers/NA.py [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+          'unknown': 0.0
+      },
+      'storage': {
+          'hydro': -10.0,
+      },
+      'source': 'mysource.com'
+    }
+    """
+
+    raw_text = get_text_from_image(session=None, link=generation_link, \
```
Comment (jarek) [inline parsers/NA.py]: Probably use `session` variable here and in fetch_exchange

Comment (systemcatch) [reply parsers/NA.py]: so session=session to avoid resetting it again?

Comment (jarek) [reply parsers/NA.py]: yeah, session=session will give get_text_from_image the same session object that fetch_production received

**Comment by systemcatch** [comment]
@corradio happy with everything here? Let me know if the tesseract installation location is a problem and we'll find a workaround.

**Comment by corradio** [comment]
All good on my side. Feel free to merge when you're good to go.

**Pull request merged**